File: matmult.cpp
#include "matmult.h"

void matmult(Matrix<double>& A, Matrix<double>& Qurmul, Matrix<double>& fh)
{
	int N = size(Qurmul, 1); int K = size(Qurmul, 2);
	
	for (int k = 1; k <= K; k++)
		for (int i = 1; i <= N; i++)
			for (int j = 1; j <= N; j++)
				fh(i, k) = fh(i, k) + A(i, j)*Qurmul(j, k);
}

File: matmult.m
function p=matmult(A, x, p)
% calcula p=p+A*x

%int: i, j, k, N, K
%matrix: A, x, p

N=size(x, 1);
K=size(x, 2);

for k=1:K
  for i=1:N
    for j=1:N
      p(i, k) = p(i, k)+A(i, j)*x(j, k);
    end
  end
end